src/docs/src/install/freebsd.rst

```diff
@@ -16,71 +16,93 @@
 
-Administrators should use ``default.ini`` as reference and only modify the
-``local.ini`` file.
+Administrators should use ``default.ini`` as a reference and only
+modify the ``local.ini`` file.
 
 Post install
-------------
-**Your installation is not complete. Be sure to complete the**
-:ref:`Setup <setup>` **steps for a single node or clustered installation.**
+============
 
-In case the install script fails to install a non-interactive user "couchdb" to
-be used for the database, the user needs to be created manually:
+**The installation is not complete. Be sure to complete the**
+:ref:`Setup <setup>` **steps for a single node or clustered
+installation.**
 
-I used the ``pw`` command to add a user "couchdb" in group "couchdb":
+Also note that the port will probably show some messages after the
+installation happened.  Make note of these instructions, although they
+can be found `in the ports tree`_ for later reference.
 
-.. code-block:: text
+In case the install script fails to install a non-interactive user
+``couchdb`` to be used for the database, the user needs to be created
+manually.
 
-    pw user add couchdb
-    pw user mod couchdb -c 'CouchDB, time to relax' -s /usr/sbin/nologin -d /var/lib/couchdb
-    pw group add couchdb
+Use the ``pw`` command from the base system to add a user ``couchdb``
+in group ``couchdb``::
 
-The user is added to ``/etc/passwd`` and should look similar to the following:
+    # pw useradd couchdb
+    # pw usermod couchdb -c 'CouchDB Account' -s /usr/sbin/nologin -d /var/empty
+    # pw groupadd couchdb
+    # pw groupmod couchdb -m couchdb
 
-.. code-block:: text
+Then the user is added to ``/etc/passwd`` and the group to
+``/etc/groups``.  These could be verified in the following way::
 
-    shell#  grep couchdb /etc/passwd
-    couchdb:*:1013:1013:Couchdb, time to relax:/var/lib/couchdb/:/usr/sbin/nologin
+    $ grep couchdb /etc/passwd
+    couchdb:*:1013:1013:CouchDB Account:/var/empty:/usr/sbin/nologin
+    $ grep couchdb /etc/groups
+    couchdb:*:1013:couchdb
 
-To change any of these settings, please refrain from editing `/etc/passwd` and
-instead use ``pw user mod ...`` or ``vipw``. Make sure that the user has no
-shell, but instead uses ``/usr/sbin/nologin``. The '*' in the second field means
-that this user can not login via password authorization. For details use
+To change any of these settings, please refrain from editing
+``/etc/passwd`` and instead use ``pw usermod ...`` or ``vipw``. Make
+sure that the user has no shell, but instead uses
+``/usr/sbin/nologin``. The ``*`` in the second field means that this
+user can not login via password authorization. For the details, see
 `man 5 passwd`_.
 
-.. _man 5 passwd: http://linux.die.net/man/5/passwd
+.. _the pre-built binary packages: https://docs.freebsd.org/en/books/handbook/ports/
+.. _rc.d service framework: https://man.freebsd.org/cgi/man.cgi?query=service&apropos=0&sektion=0&manpath=default&arch=default&format=html
```

Comment:
we can use https://man.freebsd.org/rc.d instead, much shorter

Reply:
Cool, thanks!